cli/src/com/redhat/ceylon/common/tools/AbstractTestTool.java

```diff
@@ -92,9 +92,11 @@ public void setVersion(String version) {
         this.version = version;
     }
     
-    @Option(longName = "tap")
-    @Description("Enables the Test Anything Protocol v13.")
-    public void setTap(boolean tap) {
+    @OptionArgument(argumentName = "file")
```

Comment:
Is there any use-case for printing TAP report to stdout?
I'm curious, if it wouldn't make sense to align `tap` option with `html` option and by default create file under report subfolder.